lib/std/build.zig

```diff
@@ -159,6 +160,7 @@ pub const Builder = struct {
             .is_release = false,
             .override_lib_dir = null,
             .install_path = undefined,
+            .vcpkg_root = try findVcpkgRoot(allocator),
```

Comment:
I think we should do this initialization lazily, only if a system library (other than libc) is being linked. This way the cost of vcpkg integration is not paid when an application or library is either fully self contained.

Reply:
Updated.